llvm/include/llvm/Transforms/IPO/FunctionImport.h

```diff
@@ -113,6 +113,13 @@ class FunctionImporter {
   public:
     using ImportIDTy = uint32_t;
 
+    ImportIDTable() = default;
+
+    // Three is something wrong with the application logic if we need to make a
```

Comment:
```suggestion
    // There is something wrong with the application logic if we need to make a
```

Reply:
Thanks for catching this!  Let me shorten this further down to "Something is wrong with ...".